client/coral-embed-stream/src/Embed.js

```diff
@@ -124,6 +124,16 @@ class Embed extends Component {
             <Tab>{lang.t('MY_COMMENTS')}</Tab>
             <Tab restricted={!isAdmin}>Configure Stream</Tab>
           </TabBar>
+          {
+            highlightedComment &&
+            <Button
```

Comment:
leaving myself a note here. we decided to leave this little refactor until the dust settles from the plugin additions.